docs/source/pythonapi/examples/pandas-dataframes.ipynb

```diff
@@ -2089,29 +2090,29 @@
-       "569   10003    0   10001  16  4  0   10002  10000         284     scatter   \n",
-       "570   10003    0   10001  16  3  0   10002  10000         285  absorption   \n",
-       "571   10003    0   10001  16  3  0   10002  10000         285     scatter   \n",
-       "572   10003    0   10001  16  2  0   10002  10000         286  absorption   \n",
-       "573   10003    0   10001  16  2  0   10002  10000         286     scatter   \n",
-       "574   10003    0   10001  16  1  0   10002  10000         287  absorption   \n",
-       "575   10003    0   10001  16  1  0   10002  10000         287     scatter   \n",
-       "576   10003    0   10001  16  0  0   10002  10000         288  absorption   \n",
-       "577   10003    0   10001  16  0  0   10002  10000         288     scatter   \n",
+       "    level 1        level 2            level 3        distribcell       score  \\\n",
+       "       univ   cell     lat               univ   cell                           \n",
+       "         id     id      id   x   y  z      id     id                           \n",
+       "558       0  10003   10001  16   7  0   10000  10002         279  absorption   \n",
+       "559       0  10003   10001  16   7  0   10000  10002         279     scatter   \n",
+       "560       0  10003   10001  16   8  0   10000  10002         280  absorption   \n",
+       "561       0  10003   10001  16   8  0   10000  10002         280     scatter   \n",
+       "562       0  10003   10001  16   9  0   10000  10002         281  absorption   \n",
+       "563       0  10003   10001  16   9  0   10000  10002         281     scatter   \n",
+       "564       0  10003   10001  16  10  0   10000  10002         282  absorption   \n",
+       "565       0  10003   10001  16  10  0   10000  10002         282     scatter   \n",
+       "566       0  10003   10001  16  11  0   10000  10002         283  absorption   \n",
+       "567       0  10003   10001  16  11  0   10000  10002         283     scatter   \n",
+       "568       0  10003   10001  16  12  0   10000  10002         284  absorption   \n",
+       "569       0  10003   10001  16  12  0   10000  10002         284     scatter   \n",
+       "570       0  10003   10001  16  13  0   10000  10002         285  absorption   \n",
+       "571       0  10003   10001  16  13  0   10000  10002         285     scatter   \n",
+       "572       0  10003   10001  16  14  0   10000  10002         286  absorption   \n",
+       "573       0  10003   10001  16  14  0   10000  10002         286     scatter   \n",
+       "574       0  10003   10001  16  15  0   10000  10002         287  absorption   \n",
+       "575       0  10003   10001  16  15  0   10000  10002         287     scatter   \n",
+       "576       0  10003   10001  16  16  0   10000  10002         288  absorption   \n",
+       "577       0  10003   10001  16  16  0   10000  10002         288     scatter   \n",
```

Comment:
Oh yes, I meant to make a comment about this. The new convention is correct and reflects OpenMC's lattice cell indexing; the old scheme was actually incorrect as it used OpenCG's indexing.